lib/mongoid/errors/invalid_field.rb

```diff
@@ -13,13 +13,14 @@ class InvalidField < MongoidError
       #   InvalidField.new(person, :crazy_method_name)
       #
       # @param [ Class ] klass The document class.
+      # @param [ Symbol ] field The field name.
       # @param [ Symbol ] name The method name.
-      def initialize(klass, name)
+      def initialize(klass, field, name)
```

Comment:
I know this is an API change but it is a major version... I can also make the third parameter optional...